domains/dtech.json

```diff
@@ -4,6 +4,6 @@
         "email": "brianpeter930@gmail.com"
     },
     "record": {
-        "CNAME": "dtech.is-a.dev"
+        "TXT": "_github-pages-challenge-pgwiz.dtech"
```

Comment:
yea no this wont work, refer to here on how to connect a github page https://www.is-a.dev/docs/github_pages/